apps/web/src/components/SearchRequestTable/SearchRequestsTableApi.tsx

```diff
@@ -95,9 +96,30 @@ function dateCell(date: Maybe<Scalars["DateTime"]>, intl: IntlShape) {
   ) : null;
 }
 
+const notesAccessor = (
+  searchRequest: PoolCandidateSearchRequest,
+  intl: IntlShape,
+) =>
+  searchRequest?.adminNotes ? (
+    <Spoiler
+      text={searchRequest.adminNotes}
+      linkSuffix={intl.formatMessage(
+        {
+          defaultMessage: "notes for {name}",
+          id: "I73/lv",
+          description:
+            "Link text suffix to read more notes for a pool searchRequest",
+        },
+        {
+          name: searchRequest.fullName,
```

Comment:
I'm not sure if the invisible text should be the fullName field, I would've thought request job title

Reply:
Yeah, I think they offers better context. Good suggestion 👍 

[use job title for hidden notes text](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7215/commits/02c00f620e43f373545baf421aa6786349197824)